api/response/schema.py

```diff
@@ -38,25 +42,48 @@ def mutate(self, info, **kwargs):
                 validate_rating_field(**kwargs)
                 rating = ResponseModel(rate=kwargs['rate'],
                                        room_id=kwargs['room_id'],
-                                       question_id=kwargs['question_id'])
+                                       question_id=kwargs['question_id'],
+                                       created_date=datetime.now())
                 rating.save()
                 return CreateResponse(response=rating)
         if question_type.lower() == 'check':
-            if 'check' not in kwargs:
-                raise GraphQLError("Provide a check response")
+            if 'missing_items' not in kwargs:
```

Comment:
Would it be appropriate to create a dynamic function to check the missing items in the kwargs and handle the validation, tell me what you think about this?

Reply:
yes, I think that is a good idea, this will prevent the use of nested if statement. I will do that as soon as possible

Reply:
@Bonifase I have implemented the change